### ISSUE TITLE
Unclear wording differentiating city vs. surrounding area

### ISSUE THREAD

**Issue by ProQuestionAsker** [the-pudding/music-map-story#21]
May just be me but I had to read Fact 3 of 10 a few times. For me, it reads:

> Generally, Seattle is a pop music melting pot, with a different #1 song depending on where you are. Each circle below represents the top song in a city.

The first sentence reads like there are different #1 songs within the city (in different neighborhoods, maybe?). The second sentence seems like it's talking more about the surrounding area. Maybe reword to "Generally, the Seattle area..." or something to make that clearer?

**Comment by jadiehm** [comment]
Also had some trouble with fact 3. Mine read:

> Generally, San Antonio is a pop music mono culture, with the same top song within a 45 minute drive of the city.

Which I think I would rework as something like:

> Generally, San Antonio is a pop music mono culture. You can only find a different song in the top spot if you drive 45 minutes from the city.
